spec/system/department_admin_should_be_able_to_manage_assemblies_spec.rb

```diff
@@ -57,17 +57,20 @@
 
         fill_in :assembly_slug, with: "slug"
         fill_in :assembly_hashtag, with: "#hashtag"
-        attach_file :assembly_hero_image, image1_path
-        attach_file :assembly_banner_image, image2_path
+      end
+
+      dynamically_attach_file(:assembly_hero_image, image1_path)
+      dynamically_attach_file(:assembly_banner_image, image2_path)
 
+      within ".new_assembly" do
         find("*[type=submit]").click
       end
 
       expect(page).to have_admin_callout("successfully")
       expect(Decidim::Assembly.last.area).to eq(area)
 
       within ".container" do
-        # expect(page).to have_current_path decidim_admin_assemblies.assemblies_path(parent_id: parent_assembly&.id)
+        # expect(page).to have_current_path decidim_admin_assemblies.assemblies_path(q: { parent_id_eq: parent_assembly&.id })
```

Comment:
uncomment?

Reply:
is failing...I know that we must fix but we can't run tests in local and is complicated to fix

Reply:
can you add a test that fails on 0.28 and we will check it in the future?